cmake/external/libfuzzer.cmake

```diff
@@ -27,15 +27,15 @@ if(TARGET libfuzzer)
   return()
 endif()
 
-set(tag RELEASE_601)  # trunk@{2018-07-27}
+set(tag RELEASE_601)  # latest release@{2018-07-27}
 
 ExternalProject_Add(
   libfuzzer
 
   DOWNLOAD_DIR ${FIREBASE_DOWNLOAD_DIR}
   DOWNLOAD_NAME libfuzzer
   SVN_REPOSITORY "https://llvm.org/svn/llvm-project/compiler-rt/tags/${tag}/final/lib/fuzzer"
-  LOG_DOWNLOAD FALSE    # Do not print SVN checkout messages.
+  LOG_DOWNLOAD TRUE    # Do not print SVN checkout messages.
```

Comment:
Given the change, I suspect this comment is now incorrect.

But you might just get rid of the line altogether and just let it default to ... whatever the default is. That said, I have no objection to keeping it; so if you feel that's useful, than feel free.

So I'm happy with any of:
1) Get rid of the entire line
2) Get rid of the comment (but otherwise keep the line)
3) Keep both (but in this case, ensure the comment is correct).

Reply:
I was confused by an older documentation of cmake. The new documentation for this flag is:
> When enabled, the output of the download step is logged to files.

Which means the check out messages are not printed to the console.